content/en/docs/ibm/deploy/install-kubeflow-on-iks.md

```diff
@@ -56,6 +56,17 @@ Therefore, you're recommended to set up the default storage class with Group ID
     ```shell
     kubectl patch storageclass ${OLD_STORAGE_CLASS} -p '{"metadata": {"annotations":{"storageclass.kubernetes.io/is-default-class":"false"}}}'
     ```
+### For VPC-Gen2 clusters
+
+Currently, there is no option available for setting up RWX (read-write multiple nodes) type of storage.
+RWX (read-write multiple nodes) is not a mandatory requirement to run kubeflow and most pipelines.
```

Comment:
```suggestion
RWX is not a mandatory requirement to run Kubeflow and most pipelines.
```

(RWX has been defined earlier, so this may not be necessary)